lib/index-static.jsx

```diff
@@ -39,8 +39,8 @@ function generateWithPageHTML(url, options, pageHTML) {
           return <meta key={i} name={name} content={options.meta[name]}/>;
         })}
         <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,400italic,600,700,600italic,700italic,800,800italic"/>
+        <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"/>
```

Comment:
Oh, I totally agree, I'm just saying it's not a huge deal if a **developer working locally without internet** has to see Arial instead of Open Sans, which ... actually might really annoy some developers, I suppose.

#1037 filed!